drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -328,93 +326,63 @@ void at86rf2xx_set_cca_threshold(at86rf2xx_t *dev, int8_t value)
     at86rf2xx_reg_write(dev, AT86RF2XX_REG__CCA_THRES, value);
 }
 
-void at86rf2xx_set_option(at86rf2xx_t *dev, uint16_t option, bool state)
+void at86rf2xx_set_option(at86rf2xx_t *dev,
+                          const uint16_t option, const bool state)
 {
     uint8_t tmp;
 
     DEBUG("set option %i to %i\n", option, state);
 
     /* set option field */
-    if (state) {
-        dev->netdev.flags |= option;
-        /* trigger option specific actions */
-        switch (option) {
-            case AT86RF2XX_OPT_CSMA:
+    (state) ? (dev->netdev.flags |=  option) :
+              (dev->netdev.flags &= ~option);
```

Comment:
such is used in this driver code a lot more. Further, its make the code much simpler, cleaner and (IMHO) easily readable than an equivalent `if-else` statement.